CHANGELOG.unreleased.md

```diff
@@ -21,5 +22,6 @@
 
 ### Released packages
 
+- xo-server-hooks v0.1.0
```

Comment:
We should make sure to not confused this with existing internal hooks (`start`, `stop`, `clean`).